docker/docker.go

```diff
@@ -52,11 +52,8 @@ func main() {
 		setLogLevel(logrus.InfoLevel)
 	}
 
-	// -D, --debug, -l/--log-level=debug processing
-	// When/if -D is removed this block can be deleted
 	if *flDebug {
```

Comment:
I don't think we can get rid of it because we still use the "DEBUG" env var as a global flag in the deamon